src/state_migration/tests/mod.rs

```diff
@@ -29,19 +31,63 @@ async fn test_nv17_state_migration_calibnet() -> Result<()> {
     .await
 }
 
+#[tokio::test]
+async fn test_nv18_state_migration_calibnet() -> Result<()> {
+    // State migration at height Hygge(epoch 322354) was successful,
+    // Previous state: bafy2bzacedjqwdqxlkyyuohmtcfciekl5qh2s4yf67neiuuhkibbteqoucvsm,
+    // new state: bafy2bzacedhhgkmr26rbr3yujounnz2ufiwrlvamogyabgfv6uvwq3rlv4t2i.
+    //
+    // See <https://github.com/ChainSafe/forest/actions/runs/5579505385/jobs/10195488001#step:6:515>
+    test_state_migration(
+        Height::Hygge,
+        NetworkChain::Calibnet,
+        Cid::from_str("bafy2bzacedjqwdqxlkyyuohmtcfciekl5qh2s4yf67neiuuhkibbteqoucvsm")?,
+        Cid::from_str("bafy2bzacedhhgkmr26rbr3yujounnz2ufiwrlvamogyabgfv6uvwq3rlv4t2i")?,
+    )
+    .await
+}
+
+#[tokio::test]
+async fn test_nv19_state_migration_calibnet() -> Result<()> {
+    // State migration at height Lightning(epoch 489094) was successful,
+    // Previous state: bafy2bzacedgamjgha75e7w2cgklfdgtmumsj7nadqppnpz3wexl2wl6dexsle,
+    // new state: bafy2bzacebhjx4uqtg6c65km46wiiq45dbbeckqhs2oontwdzba335nxk6bia.
+    //
+    // See <https://github.com/ChainSafe/forest/actions/runs/5579505385/jobs/10195488001#step:6:232>
+    test_state_migration(
+        Height::Lightning,
+        NetworkChain::Calibnet,
+        Cid::from_str("bafy2bzacedgamjgha75e7w2cgklfdgtmumsj7nadqppnpz3wexl2wl6dexsle")?,
+        Cid::from_str("bafy2bzacebhjx4uqtg6c65km46wiiq45dbbeckqhs2oontwdzba335nxk6bia")?,
+    )
+    .await
+}
+
 async fn test_state_migration(
     height: Height,
     network: NetworkChain,
     old_state: Cid,
     expected_new_state: Cid,
 ) -> Result<()> {
-    // TODO: CompressedCarV1BackedBlockstore does not work for nv18 and nv19,
-    // use UncompressedCarV1BackedBlockstore instead
+    // Car files are cached under data folder for Go test to pick up without network access
+    let car_path = format!("./src/state_migration/tests/data/{old_state}.car");
+    if !Path::new(&car_path).is_file() {
+        let tmp: tempfile::TempPath = tempfile::NamedTempFile::new()?.into_temp_path();
+        {
+            let mut reader = crate::utils::net::reader(&format!(
+                " https://forest-snapshots.fra1.cdn.digitaloceanspaces.com/state/{old_state}.car"
```

Comment:
Does it belong in the snapshots bucket? We can create new ones freely. How about a `forest-continuous-integration`?

Reply:
I can do that

Reply:
Let's do it. I'd rather separate our "prod" buckets from dev ones.

Reply:
Fixed